### PR TITLE
Merge `GistComment` type stub back to source

### PR DESCRIPTION
**Pull request by trim21**

### CODE REVIEW

**Review by Borda — APPROVED** [review]

File: github/GistComment.py [hunk 1]
```diff
@@ -27,81 +27,71 @@
...
+    def url(self) -> str:
         self._completeIfNotSet(self._url)
         return self._url.value
 
     @property
-    def user(self):
-        """
-        :type: :class:`github.NamedUser.NamedUser`
-        """
+    def user(self) -> github.NamedUser.NamedUser:
```
Comment (EnricoMi) [inline github/GistComment.py:82]:
```suggestion
    def user(self) -> NamedUser:
```

File: github/GistComment.py [hunk 2]
```diff
@@ -27,81 +27,71 @@
...
     This class represents GistComments. The reference can be found here https://docs.github.com/en/rest/reference/gists#comments
     """
 
+    def _initAttributes(self) -> None:
+        self._body: Attribute[str] = NotSet
+        self._created_at: Attribute[datetime] = NotSet
+        self._id: Attribute[int] = NotSet
+        self._updated_at: Attribute[datetime] = NotSet
+        self._url: Attribute[str] = NotSet
+        self._user: Attribute[github.NamedUser.NamedUser] = NotSet
```
Comment (EnricoMi) [inline github/GistComment.py:51]:
```suggestion
        self._user: Attribute[NamedUser] = NotSet
```

**Review by EnricoMi — CHANGES_REQUESTED** [review]
minor cleanups

File: github/GistComment.py (hunk 1 above)
Comment (trim21) [reply github/GistComment.py:82]: I would suggest no.

let's discuss https://github.com/PyGithub/PyGithub/pull/2727 first.

**Review by EnricoMi — APPROVED** [review]
LGTM!

**Pull request merged**